languages.toml

```diff
@@ -1,6 +1,145 @@
 # Language support configuration.
 # See the languages documentation: https://docs.helix-editor.com/master/languages.html
 
+[language-server]
+
+taplo = { command = "taplo", args = ["lsp", "stdio"] }
+elixir-ls = { command = "elixir-ls", config = { elixirLS.dialyzerEnabled = false } }
+mint = { command = "mint", args = ["ls"] }
+vscode-json-language-server =  { command = "vscode-json-language-server", args = ["--stdio"], config = { provideFormatter = true } }
+clangd = { command = "clangd" }
+omnisharp = { command = "OmniSharp", args = [ "--languageserver" ] }
+vscode-css-language-server = { command = "vscode-css-language-server", args = ["--stdio"], config = { "provideFormatter" = true }}
+vscode-html-language-server = { command = "vscode-html-language-server", args = ["--stdio"], config = { provideFormatter = true } }
+pylsp = { command = "pylsp" }
+nls = { command = "nls" }
+nil = { command = "nil" }
+solargraph = { command = "solargraph", args = ["stdio"] }
+bash-language-server = { command = "bash-language-server", args = ["start"] }
+intelephense = { command = "intelephense", args = ["--stdio"] }
```

Comment:
Would it be better to sort this list alphabetically?

Reply:
Yeah maybe it's better to sort it (I kind of used the order of the languages for the language servers) and use the long-form (at least something to the level `[language-server.<name>]`). Anyone else agreeing here or has some suggestions?